app/views/contexts/calculated_summary_context.py

```diff
@@ -70,11 +68,6 @@ def build_view_context_for_calculated_summary(self, current_location):
                 "title": block.get("title") % dict(total=formatted_total),
                 "collapsible": block.get("collapsible", False),
                 "summary_type": "CalculatedSummary",
-                "headers": [
```

Comment:
How comes headers are removed throughout?. How does the DS handle this now?. How will it have context?.